### ISSUE TITLE
Include DDS.js

### ISSUE THREAD

**Issue by FRosner** [FRosner/spawncamping-dds#289]
@Gerrrr do you think we can already include it in the release candidate for 4.0.0? I would love to, as I am going to release 4.0.0-beta today and I am having a few issues left in the front-end anyway that would probably be fixed by your library...

**Comment by Gerrrr** [comment]
Yep, I am going to submit a PR with DDS.js integration later this week.
